core/scripts/go.mod

```diff
@@ -24,7 +24,7 @@ require (
 	github.com/prometheus/client_golang v1.17.0
 	github.com/shopspring/decimal v1.3.1
 	github.com/smartcontractkit/chainlink-automation v1.0.4
-	github.com/smartcontractkit/chainlink-common v0.1.7-0.20240614134652-1fb0b48758af
+	github.com/smartcontractkit/chainlink-common v0.1.7-0.20240617160144-fa92852ab5d9
```

Comment:
Mind bumping to latest while you're at it?

Reply:
tidy keeps it at  this version